interfaces/src/db.ts

```diff
@@ -1,6 +1,12 @@
+import { Patch } from './subscriptions';
+
 export type Comparable = string | number | Date;
 export type Equatable = Comparable | boolean;
 
+// Typescript's way of defining any - undefined
+// see: https://github.com/Microsoft/TypeScript/issues/7648
+export type Serializable = {} | null;
```

Comment:
Does this generate the correct JSON schema?

Reply:
Then it is not correct (you could call it a bug in typescript-json-schema).
We should probably just use `any` for the wire protocol.
If we can't  we could try to change the type to `{} | Array | string | number | boolean | null`.